### DISCUSSION TITLE
Skiff is acquired by Notion, and sunsetting its ENS email service.

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#118] [Ideas]
Not good news, any alternative suggestions?

https://github.com/orgs/dcdao/discussions/66

**Comment by hackfisher** [comment]
https://ethermail.io/ works for me, but encountered some issues when testing it using ENS subdomain.

**Reply by 1022ranji** [reply]
More info:
Ethermail replied that subdomain can be used, but only if you first register/log in Ethermail account with ENS domain and then set the subdomain in it.
Since DCDAO's ENS domain is purchased with a multi-signature account, according to the current situation, Ethermail doesn't support logging in with a multi-signature account directly (I'm waiting for their reply and confirmation on this point).

**Comment by hackfisher** [comment]
Migrating to dcdao.box

https://github.com/orgs/dcdao/discussions/120
